core/liquidity/v2/sla.go

```diff
@@ -88,7 +89,7 @@ func (e *Engine) calculateCurrentTimeBookFraction(now, start time.Time, s time.D
 		timeBookFraction = num.DecimalFromInt64(s.Nanoseconds()).Div(num.DecimalFromInt64(lNano))
 	}
 
-	return timeBookFraction
+	return num.MinD(num.DecimalOne(), timeBookFraction)
```

Comment:
are we sure we want to keep this? If the value > 1, wouldn't that imply that the party hasn't met the commitment last time around, and therefore the fraction would _have_ to be < 1

Reply:
I'm not sure I follow. It should always be that `timeBookFraction <= 1` and if its not something has gone wrong so the cap is just there for safety.